<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Uplaod_Files_Dialog</class>
 <widget class="QWidget" name="Uplaod_Files_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Files To Upload</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item row="0" column="0">
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Upload Time:</string>
        </property>
       </widget>
       <widget class="QDateTimeEdit" name="dateTimeEdit"/>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Upload now</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QListWidget" name="listWidget"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
